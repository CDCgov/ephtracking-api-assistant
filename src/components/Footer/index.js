import React from 'react';

const Footer = () => (
  <footer className="footer">
    <div className="container">
      <div className="content has-text-centered">
        <p>
          <strong>Environmental Public Health Trackign API Assistant</strong> by <a href="https://github.com/stfnh">Stefan Hochwarter</a>.
          The source code is licensed <a href="http://opensource.org/licenses/mit-license.php">MIT</a>.
        </p>
        <a class="github-button" href="https://github.com/stfnh/ephtracking-api-assistant/fork" data-size="large" aria-label="Fork stfnh/ephtracking-api-assistant on GitHub">Fork</a>      </div>
    </div>
  </footer>
);

export default Footer;
