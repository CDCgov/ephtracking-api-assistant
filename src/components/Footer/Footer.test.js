import React from 'react';
import Footer from './';
import renderer from 'react-test-renderer';

describe('Footer', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
