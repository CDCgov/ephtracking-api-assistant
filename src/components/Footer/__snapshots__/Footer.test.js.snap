// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footer
  className="footer"
>
  <div
    className="container"
  >
    <div
      className="content has-text-centered"
    >
      <p>
        <strong>
          Environmental Public Health Trackign API Assistant
        </strong>
         by
         
        <a
          href="https://github.com/stfnh"
        >
          Stefan Hochwarter
        </a>
        . code is licensed
         
        <a
          href="http://opensource.org/licenses/mit-license.php"
        >
          MIT
        </a>
        .
      </p>
      <a
        aria-label="Fork stfnh/ephtracking-api-assistant on GitHub"
        className="github-button"
        data-size="large"
        href="https://github.com/stfnh/ephtracking-api-assistant/fork"
      >
        Fork
      </a>
       
    </div>
  </div>
</footer>
`;
